### PR TITLE
Update messageEvents.md

### PR DESCRIPTION
**Pull request by ajrice6713**
Added warning about checking direction of message in callback

## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Review by jchavez443 — APPROVED** [review]

File: messaging/callbacks/messageEvents.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Bandwidth uses HTTP Callbacks (also known as [webhooks](https://webhooks.pbworks
 * You will get a callback for any event related to that message.
   * For example, you will get an HTTP callback when your message is delivered, or blocked. In addition, you will get an event for any kind of Delivery Receipt that the destination carrier sends back, regarding the delivery of your message.
 * For each message sent, you **will** receive either (but not both) a [Message Delivered](msgDelivered.md) or [Message Failed](messageFailed.md) event.
+  * It is essential to check the direction of the message in the callback you receive as not to handle to the outbound callback message as a new incoming text message and generate an auto response. This could create a loop of messages being sent from your account as you continuously respond to outbound message callbacks.
```
Comment (jmulford-bw) [inline messaging/callbacks/messageEvents.md]: I'd reword this like this

`It is essential to check the direction of the message in the callback you receive. For example, if your use case depends on responding to inbound messages, you do not want to respond to an outbound message callback. This could create a loop of messages being sent from your account as you continuously respond to outbound message callbacks.`

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

File: messaging/callbacks/messageEvents.md (hunk 1 above)
Comment (ajrice6713) [reply messaging/callbacks/messageEvents.md]: Changed - i like that better

**Review by ajrice6713 — COMMENTED** [review]
Changes made as requested
